cartoframes/data/observatory/catalog/dataset.py

```diff
@@ -375,7 +375,7 @@ def _join_geographies_geodataframes(geographies_gdf1, geographies_gdf2):
         return join_gdf['id'].unique()
 
     @check_do_enabled
-    def to_csv(self, file_path, credentials=None, limit=None):
+    def to_csv(self, file_path, credentials=None, limit=None, order_by=None):
```

Comment:
is the `order_by` required or we're adding more to the scope?

Reply:
ok, fairly easy to implement and improves testability :ok_hand:

Reply:
Yep, that is, it is "out of scope", but I want to ensure e2e tests (and fix them because they are failing now)